roles/ansible-test/defaults/main.yaml

```diff
@@ -14,3 +14,4 @@ ansible_test_collection_dir: ~/.ansible/collections/ansible_collections
 ansible_test_enable_ara: false
 ansible_test_split_in: 1
 ansible_test_do_number: 1
+ansible_coverage_report: ""
```

Comment:
lets default this in playbooks to empty, so we don't have to update all jobs for it